liiweb/templates/peachjam/_social_media.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n static %}
+<div class="col-12 col-md mb-2">
```

Comment:
Rule of thumb: a `.col` must always be a direct child of a `.row` - so if you can't guarantee that, then there's something up.

So, this line is contextual - it depends on how this file is included. This is only necessary when it's included in the footer. So remove it from here, and just have the h5 onwards.